### PR TITLE
Nopatch CVE-2019-18276 in bash

### PR DESCRIPTION
**Pull request by thcrain-msft**
###### Merge Checklist  
**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Nopatch CVE-2019-18276 in bash

###### Change Log  

- Fixes CVE-2019-18276

###### Does this affect the toolchain?  

**YES**

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2019-18276

###### Test Methodology

- Local toolchain build
- Build of a sample of packages that depend on bash

### CODE REVIEW

**Comment by thcrain-msft** [comment]
Note: Passes linting, but a bug in the action causes it to fail.

File: SPECS/bash/bash.spec [hunk 1]
```diff
@@ -1,45 +1,51 @@
...
+# Backporting the patch is non-trivial, as well.
+Patch1:         CVE-2019-18276.nopatch
 BuildRequires:  readline
 Requires:       readline
-Requires(post):    /bin/grep
-Requires(post):    /bin/cp
-Requires(postun):  /bin/grep
-Requires(postun):  /bin/mv
+Requires(post): /bin/cp
+Requires(post): /bin/grep
```
Comment (PawelWMS) [inline SPECS/bash/bash.spec:20]: Nit: could we align these `Requires`?

**Review by PawelWMS — APPROVED** [review]

**Review by christopherco — APPROVED** [review]
:shipit:

File: SPECS/bash/bash.spec (hunk 1 above)
Comment (thcrain-msft) [reply SPECS/bash/bash.spec:20]: Nit granted!

**Pull request merged**